### PR TITLE
reference: update pd-ctl flags

### PR DESCRIPTION
**Pull request by lhy1024**
### What is changed, added or deleted? 

add pd control flags document and sort flags by short option dictionary order.

### What is the related PR or file link(s)?

https://github.com/pingcap/pd/pull/1763

### Which version does your change affect? 

dev 3.0

### CODE REVIEW

File: dev/reference/tools/pd-control.md [hunk 1]
```diff
@@ -49,32 +49,42 @@ export PD_ADDR=http://127.0.0.1:2379 &&
...
-- 默认值: ""
++ 输出帮助信息
++ 默认值: false
 
-### --cert
+### \-\-interact,-i
 
-- 指定 PEM 格式的 SSL 证书文件路径
-- 默认值: ""
++ 使用交互模式(进入 readline)
```
Comment (TomShawn) [inline dev/reference/tools/pd-control.md]:
```suggestion
+ 使用交互模式（进入 readline）
```

File: dev/reference/tools/pd-control.md [hunk 2]
```diff
@@ -49,32 +49,42 @@ export PD_ADDR=http://127.0.0.1:2379 &&
 
 ## 命令行参数(flags)
 
-### \-\-pd,-u
+### --cacert
 
-+ 指定 PD 的地址
-+ 默认地址: `http://127.0.0.1:2379`
-+ 环境变量: PD_ADDR
+- 指定 PEM 格式的受信任 CA 的证书文件路径
```
Comment (TomShawn) [inline dev/reference/tools/pd-control.md]:
```suggestion
- 指定 PEM 格式的受信任 CA 证书的文件路径
```

File: dev/reference/tools/pd-control.md [hunk 3]
```diff
@@ -49,32 +49,42 @@ export PD_ADDR=http://127.0.0.1:2379 &&
...
+- 默认值: ""
+
+### --cert
+
+- 指定 PEM 格式的 SSL 证书文件路径
```
Comment (TomShawn) [inline dev/reference/tools/pd-control.md]:
```suggestion
- 指定 PEM 格式的 SSL 证书的文件路径
```

File: dev/reference/tools/pd-control.md [hunk 4]
```diff
@@ -49,32 +49,42 @@ export PD_ADDR=http://127.0.0.1:2379 &&
...
 ### --key
 
 - 指定 PEM 格式的 SSL 证书密钥文件路径，即 `--cert` 所指定的证书的私钥
 - 默认值: ""
 
+### \-\-pd,-u
+
++ 指定 PD 的地址
++ 默认地址: `http://127.0.0.1:2379`
++ 环境变量: PD_ADDR
```
Comment (TomShawn) [inline dev/reference/tools/pd-control.md]:
```suggestion
+ 环境变量：`PD_ADDR`
```

File: v2.1/reference/tools/pd-control.md [hunk 5]
```diff
@@ -62,11 +51,22 @@ export PD_ADDR=http://127.0.0.1:2379
 - 指定 PEM 格式的 SSL 证书文件路径
 - 默认值: ""
 
+### \-\-detach,-d
+
++ 使用单命令行模式(不进入 readline)
```
Comment (TomShawn) [inline v2.1/reference/tools/pd-control.md]:
```suggestion
+ 使用单命令行模式（不进入 readline）
```

File: v2.1/reference/tools/pd-control.md [hunk 6]
```diff
@@ -62,11 +51,22 @@ export PD_ADDR=http://127.0.0.1:2379
...
 ### --key
 
 - 指定 PEM 格式的 SSL 证书密钥文件路径，即 `--cert` 所指定的证书的私钥
 - 默认值: ""
 
+### \-\-pd,-u
+
++ 指定 PD 的地址
++ 默认地址: `http://127.0.0.1:2379`
++ 环境变量: PD_ADDR
```
Comment (TomShawn) [inline v2.1/reference/tools/pd-control.md]:
```suggestion
+ 环境变量：`PD_ADDR`
```

File: v2.1/reference/tools/pd-control.md [hunk 7]
```diff
@@ -62,11 +51,22 @@ export PD_ADDR=http://127.0.0.1:2379
...
+
 ### --key
 
 - 指定 PEM 格式的 SSL 证书密钥文件路径，即 `--cert` 所指定的证书的私钥
 - 默认值: ""
 
+### \-\-pd,-u
+
++ 指定 PD 的地址
++ 默认地址: `http://127.0.0.1:2379`
```
Comment (TomShawn) [inline v2.1/reference/tools/pd-control.md]:
```suggestion
+ 默认地址：`http://127.0.0.1:2379`
```

File: v2.1/reference/tools/pd-control.md [hunk 8]
```diff
@@ -62,11 +51,22 @@ export PD_ADDR=http://127.0.0.1:2379
 - 指定 PEM 格式的 SSL 证书文件路径
 - 默认值: ""
 
+### \-\-detach,-d
+
++ 使用单命令行模式(不进入 readline)
++ 默认值: false
```
Comment (TomShawn) [inline v2.1/reference/tools/pd-control.md]:
```suggestion
+ 默认值：false
```

File: dev/reference/tools/pd-control.md [hunk 9]
```diff
@@ -49,32 +49,42 @@ export PD_ADDR=http://127.0.0.1:2379 &&
...
 ## 命令行参数(flags)
 
-### \-\-pd,-u
+### --cacert
 
-+ 指定 PD 的地址
-+ 默认地址: `http://127.0.0.1:2379`
-+ 环境变量: PD_ADDR
+- 指定 PEM 格式的受信任 CA 的证书文件路径
+- 默认值: ""
```
Comment (TomShawn) [inline dev/reference/tools/pd-control.md]:
```suggestion
- 默认值：""
```

File: dev/reference/tools/pd-control.md [hunk 10]
```diff
@@ -49,32 +49,42 @@ export PD_ADDR=http://127.0.0.1:2379 &&
...
+
+### --cert
+
+- 指定 PEM 格式的 SSL 证书文件路径
+- 默认值: ""
```
Comment (TomShawn) [inline dev/reference/tools/pd-control.md]:
```suggestion
- 默认值：""
```

File: dev/reference/tools/pd-control.md [hunk 11]
```diff
@@ -49,32 +49,42 @@ export PD_ADDR=http://127.0.0.1:2379 &&
...
 + 使用单命令行模式(不进入 readline)
 + 默认值: true
 
-### --cacert
+### \-\-help,-h
 
-- 指定 PEM 格式的受信任 CA 的证书文件路径
-- 默认值: ""
++ 输出帮助信息
++ 默认值: false
```
Comment (TomShawn) [inline dev/reference/tools/pd-control.md]:
```suggestion
+ 默认值：false
```

File: dev/reference/tools/pd-control.md [hunk 12]
```diff
@@ -49,32 +49,42 @@ export PD_ADDR=http://127.0.0.1:2379 &&
...
 
-### --cert
+### \-\-interact,-i
 
-- 指定 PEM 格式的 SSL 证书文件路径
-- 默认值: ""
++ 使用交互模式(进入 readline)
++ 默认值: false
```
Comment (TomShawn) [inline dev/reference/tools/pd-control.md]:
```suggestion
+ 默认值：false
```

File: dev/reference/tools/pd-control.md [hunk 13]
```diff
@@ -49,32 +49,42 @@ export PD_ADDR=http://127.0.0.1:2379 &&
...
 
 ### --key
 
 - 指定 PEM 格式的 SSL 证书密钥文件路径，即 `--cert` 所指定的证书的私钥
 - 默认值: ""
 
+### \-\-pd,-u
+
++ 指定 PD 的地址
++ 默认地址: `http://127.0.0.1:2379`
```
Comment (TomShawn) [inline dev/reference/tools/pd-control.md]:
```suggestion
+ 默认地址：`http://127.0.0.1:2379`
```

File: v3.0/reference/tools/pd-control.md [hunk 14]
```diff
@@ -71,11 +60,27 @@ export PD_ADDR=http://127.0.0.1:2379 &&
 - 指定 PEM 格式的 SSL 证书文件路径
 - 默认值: ""
 
+### \-\-detach,-d
+
++ 使用单命令行模式(不进入 readline)
```
Comment (TomShawn) [inline v3.0/reference/tools/pd-control.md]:
```suggestion
+ 使用单命令行模式（不进入 readline）
```

File: v3.0/reference/tools/pd-control.md [hunk 15]
```diff
@@ -71,11 +60,27 @@ export PD_ADDR=http://127.0.0.1:2379 &&
...
++ 默认值: true
```
Comment (TomShawn) [inline v3.0/reference/tools/pd-control.md]:
```suggestion
+ 默认值：true
```

File: v3.0/reference/tools/pd-control.md [hunk 16]
```diff
@@ -71,11 +60,27 @@ export PD_ADDR=http://127.0.0.1:2379 &&
...
+
+### \-\-interact,-i
+
++ 使用交互模式(进入 readline)
```
Comment (TomShawn) [inline v3.0/reference/tools/pd-control.md]:
```suggestion
+ 使用交互模式（进入 readline）
```

File: v3.0/reference/tools/pd-control.md [hunk 17]
```diff
@@ -71,11 +60,27 @@ export PD_ADDR=http://127.0.0.1:2379 &&
...
++ 默认值: false
```
Comment (TomShawn) [inline v3.0/reference/tools/pd-control.md]:
```suggestion
+ 默认值：false
```

File: v3.0/reference/tools/pd-control.md [hunk 18]
```diff
@@ -71,11 +60,27 @@ export PD_ADDR=http://127.0.0.1:2379 &&
...
+
 ### --key
 
 - 指定 PEM 格式的 SSL 证书密钥文件路径，即 `--cert` 所指定的证书的私钥
 - 默认值: ""
 
+### \-\-pd,-u
+
++ 指定 PD 的地址
++ 默认地址: `http://127.0.0.1:2379`
```
Comment (TomShawn) [inline v3.0/reference/tools/pd-control.md]:
```suggestion
+ 默认地址：`http://127.0.0.1:2379`
```

File: v3.0/reference/tools/pd-control.md [hunk 19]
```diff
@@ -71,11 +60,27 @@ export PD_ADDR=http://127.0.0.1:2379 &&
...
++ 环境变量: PD_ADDR
```
Comment (TomShawn) [inline v3.0/reference/tools/pd-control.md]:
```suggestion
+ 环境变量：`PD_ADDR`
```

**Comment by lilin90** [comment]
@TomShawn PTAL again~ Thanks!

**Comment by lhy1024** [comment]
@rleungx PTAL

**Comment by lilin90** [comment]
@dcalvin FYI in case you need to update tikv.org.

**Comment by lhy1024** [comment]
@TomShawn PTAL

**Comment by TomShawn** [comment]
LGTM

**Pull request merged**